install_sct

```diff
@@ -22,6 +22,7 @@
 #  -d   Prevent the (re)-installation of the data/ directory
 #  -b   Prevent the (re)-installation of the SCT binaries files
 #  -c   Prevent checks from being run to validate the installation
+#  -g   Specify git ref (branch, tag, commit SHA) to clone when 'install_sct' is run by itself
```

Comment:
From https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4049#discussion_r1117229028:

> Running the install script standalone within CI workflows

Reminder note to self to update the CI to use the standalone installation method, as a way of testing out the changes I've made.

Reply:
Ah, I was mistaken here! We don't actually use the _["standalone installation method, specifying a PR git ref"]_ installation in our CI anymore, because it used to break PRs from forks:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/33e134edf001670725d5fb36a06b8f37245d9580/.github/workflows/tests.yml#L387-L393

- https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3779

So... we don't actually need a `-g` option at all?

Reply:
Worryingly, this means we're not testing the "standalone installation" method at all in our CI... hmm...